app/views/publishers/organisations/show.html.slim

```diff
@@ -2,8 +2,8 @@
 
 .govuk-grid-row
   .govuk-grid-column-full
-    = render(Publishers::VacanciesComponent.new(organisation: @organisation, sort: @sort, selected_type: @selected_type, filters: @filters, filters_form: @managed_organisations_form, sort_form: @sort_form))
-    = render(Publishers::NoVacanciesComponent.new(organisation: @organisation))
+    = render(Publishers::VacanciesComponent.new(organisation: @organisation, sort: @sort, selected_type: @selected_type, filters: @filters, filters_form: @managed_organisations_form, sort_form: @sort_form, email: @publisher_email))
```

Comment:
Can we just use `current_publisher.email` here if it's a helper method?

Reply:
`current_jobseeker` is a helper method provided by devise. Will add a helper method to be accessible by the views for `current_publisher` too